src/Game/GameObjects/PlayerMobile.cs

```diff
@@ -1860,6 +1867,34 @@ private bool IsDoor(ushort type)
                    || type >= 0x9B3C && type <= 0x9B4B;
         }
 
+        private bool IsSecretDoor(ushort type)
```

Comment:
this code can me merge into the IsDoor function.
Basically if the Profile.SecretDoor option is false
```csharp
bool IsDoor(Graphic g)
{
    // the issecretdoor behaviour
    ...
   // end of issecretdoor behaviour

if (engine.profile.current.issecretdoor)
{
       // the standard isdoor behaviour
    ...
   // end of standard isdoor behaviour
 
}
 return false;
}
```